### ISSUE TITLE
[release request] OpenTelemetry.Instrumentation.AWS stable release

### ISSUE THREAD

**Issue by ciolial** [open-telemetry/opentelemetry-dotnet-contrib#2346]
### Component

OpenTelemetry.Instrumentation.AWS

### Version

1.1.0

### Additional context

Hi,
Sorry if I'm not using the right issue type. Currently I'm manually instrumenting a lambda using the packages `OpenTelemetry.Instrumentation.AWS v1.1.0-beta.6` and `OpenTelemetry.Instrumentation.AWSLambda v1.3.0-beta.1` with quite good results. Now I'd like to bring the lambda in the production environment but I have some doubts regarding these beta versions. Is there an idea to release a stable version of these packages?

Thank you.

**Comment by opentelemetrybot** [comment]
@srprash @ppittle @muhammad-othman @rypdal @Oberon00 @open-telemetry/dotnet-approvers @open-telemetry/dotnet-maintainers

Post a comment with "/PrepareRelease" in the body if you would like me to execute the prepare release script for the component and version listed in the description.

**Comment by opentelemetrybot** [comment]
@srprash @ppittle @muhammad-othman @rypdal @Oberon00 @open-telemetry/dotnet-approvers @open-telemetry/dotnet-maintainers

Post a comment with "/PrepareRelease" in the body if you would like me to execute the prepare release script for the component and version listed in the description.

**Comment by ppittle** [comment]
@ciolial - thank you for the feedback.  I'm actively working towards a stable release for OpenTelemetry.Instrumentation.AWS and the other OpenTelemetry.*.AWS packages.

Once #2437 is merged, I'll release a Release Candidate.  That will need to bake for ~3 weeks so we can collect any feedback.  Depending on feedback, the next step will be a GA (stable) release.

**Comment by ppittle** [comment]
Update: There is a Release Candidate published to nuget: https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AWS/1.10.0-rc.1

We will wait until at least Jan 27 to collect user feedback and evaluate any issues, and then will publish a stable GA release.

@ciolial - If you have an opportunity to test the Release Candidate, any feedback would be most welcome!

**Comment by ppittle** [comment]
Stable release requested in #2520 to be merged in #2521

**Comment by ppittle** [comment]
Stable packages have been released.

This issue can be closed.

**Comment by ciolial** [comment]
Hi @ppittle, I have been busy these weeks. Thank you for the releases. I'm going to use them on our lambda in a prod env.
I'll let you know any feedback.

Thanks again.

**Comment by ppittle** [comment]
@ciolial - great to hear!  If you have any feedback I'd love to hear it.  Either create a new Issue in this repo, or you can DM me in the [OpenTelemetry Slack](https://opentelemetry.io/community/end-user/slack-channel/)

**Comment by ppittle** [comment]
@ciolial - since this issue is already closed, can you create a new issue for the `System.MissingMethodException`?

And if possible, can you include a minimal sample that can reproduce the problem?
